docs/conf.py

```diff
@@ -514,27 +513,3 @@ def get_version(*file_paths):
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3.12', None),
 }
-
-
-def on_init(app):  # pylint: disable=unused-argument
-    """
-    Run sphinx-apidoc after Sphinx initialization.
```

Comment:
1. Are swagger docs completely separate from all this?
2. How does the LMS generate these docs? https://docs.openedx.org/projects/edx-platform/en/latest/references/lms_apis.html. Wouldn't we want something similar for this service?

Reply:
Swagger is separate, yeah. I'll add that to https://github.com/openedx/codejail-service/issues/3 for future work.